src/room/InCallView.tsx

```diff
@@ -200,8 +200,9 @@ export const InCallView: FC<InCallViewProps> = ({
   );
 
   const onLeavePress = useCallback(() => {
+    livekitRoom.disconnect();
     onLeave();
```

Comment:
What about moving this call into `onLeave`? Would it make sense for it to live there?

Reply:
This is passed in from GroupCallView where we don't have the livekit room object (and is also called on a leave due to a disconnect) - commented.